ONSdigital/eq-author
# drop zone fix - make a default for the drop zone and expand the catch…

__What__
Drop zone fixes
1. Expanded drop zone
2. Default text for each section when empty

__How to test__
1. Create a survey and questionnaire
2. Check that the automatic section has default dropzone text when empty
3. New drop zones have default text and are independent when empty
4. That the dropzone now is the size of the box

__Who can test__
Anyone but @LJBabbage
